librz/il/theory_bool.c

```diff
@@ -4,28 +4,28 @@
 #include <rz_il/rzil_opcodes.h>
 #include <rz_il/rzil_vm.h>
 
-void *rz_il_handler_b0(RzILVM *vm, RzILOp *op, RzILOpArgType *type) {
+void *rz_il_handler_bool_false(RzILVM *vm, RzILOp *op, RzILOpArgType *type) {
```

Comment:
Or, we could use these new names but add a comment to which BAP CT operation it corresponds.